src/features/home/components/mapSwiper/MapSwiper.tsx

```diff
@@ -16,21 +14,13 @@ const settings = {
 };
 
 interface MapSwiperProps extends PropsWithChildren {
-  currentPosition: number | null;
+  currentPage: number | null;
 }
 
-export const MapSwiper = ({ currentPosition, children }: MapSwiperProps) => {
-  const [initialSlide, setInitialSlide] = useState<number | null>(null);
-
-  useEffect(() => {
-    if (currentPosition) {
-      setInitialSlide(Math.floor(currentPosition / GOAL_COUNT_PER_PAGE));
-    }
-  }, [currentPosition]);
-
+export const MapSwiper = ({ currentPage, children }: MapSwiperProps) => {
```

Comment:
홍석넴 이거 위에서 PropsWithChildren 상속 받기보다

```suggestion
export const MapSwiper = ({ currentPage, children }: PropsWithChildren<MapSwiperProps>) => {
```
도 가능하답니다잇 (별건 아니여서 시간 있을 때 😉

Reply:
반영하긴 했는데 이렇게 하면 좋은 점이 있나요~?? 👀 잘모르겠어서 👀👀